src/components/RenderBooks.js

```diff
@@ -1,21 +1,36 @@
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import Button from './Button';
+import { removeBook } from '../redux/books/myBookSlice';
+
+function Render({
+  renderData,
+}) {
+  const dispatch = useDispatch();
+
+  const removedBooks = (item_Id) => {
+    dispatch(removeBook(item_Id));
+  };
+  return (
+    <>
+      {renderData.map((item) => (
+        <div id={item.item_Id} key={item.item_Id}>
+          <h2>{item.title}</h2>
+          <p>
+            By:
+            <br />
+            {item.author}
+          </p>
+          <Button value="Remove" className="remove-btn" type="button" onClick={(e) => removedBooks(e.target.parentElement.id)} />
```

Comment:
- Optional
Instead of passing `parentElement.id` into the `removedBooks( )`, I would strong recommend passing `item_Id` since you already have access to it. You should never rely on the selector elements and instead work on the data which you have at hand.

Reply:
Thanks, bro for your suggestion, I have implemented your optional suggestion